accounts/admin.py

```diff
@@ -5,16 +5,19 @@
 """
 Registering user model to the admin and extending the UserAdmin so that the user admin page looks like the default user admin panel
 List_display specifies the attributes that will appear on the admin page.
-add_fieldsets specifies the attributes to be using when creating a user on the admin panel
+add_fieldsets specifies the attributes to be using when creating a user on the admin panel.
+fieldsets specifies the additional user fields
 """
 @admin.register(User)
 class CustomUserAdmin(UserAdmin):
-    list_display = ('email', 'is_staff', 'is_active',)
+    list_display = ('username', 'email', 'is_staff', 'is_active',)
+    fieldsets = (
+        (None, {'fields': ('username', 'password')}),
+        ('Personal info', {'fields': ('email', 'first_name', 'last_name', 'phone_number')}),
```

Comment:
fieldsets displays fields when updating a user

Reply:
I tried updating the user information but I didn't see the phone number field.

Reply:
Would you paste your code here?

Reply:
now i got it

Reply:
This can be done by adding the phone number in the REQUIRED FIELDS...in the custom user model implementation..
I did not include it because I thought that would be too many fields to add when creating superuser and assumed the phone number would be added when updating the superuser... that's my view.

If it's important I can change and add it to the required fields.